### ISSUE TITLE
How Can i show cross icon using click instated of right click. when selected node

### ISSUE THREAD

**Issue by parthsavaliya1** [jerosoler/Drawflow#843]
I want to show cross icon when user select node using click instead of right click

**Comment by jerosoler** [comment]
You can try something like:

- https://github.com/jerosoler/Drawflow/issues/235

**Comment by parthsavaliya1** [comment]
Yes. this is right but cross icon also shown when user click on right click. cross icon should not be shown when user click on right click.

**Comment by parthsavaliya1** [comment]
How can i disable double click and right click only on connection line.

**Comment by parthsavaliya1** [comment]
@jerosoler is there any solution for this?
